R/db_create_coverage_table.R

```diff
@@ -1,21 +1,27 @@
-#' Create coverage table
+#' Create coverage tables
 #'
-#' Create a table with coverage estimates at regional, WLD and TOT levels.
+#' Create a list of tables with coverage estimates at 1) regional, WLD, TOT and
+#' 2) income group levels.
 #'
 #' @param ref_year_table data.table: Full interpolated means table. Output of
 #'   `db_create_ref_year_table()`.
+#' @param cl_table data.table: Country list table with all WDI countries.
+#' @param incgrp_table data.table: Table with historical income groups for all WDI countries.
 #' @inheritParams db_create_ref_year_table
 #' @param digits numeric: The number of digits the returned coverage numbers are
 #'   rounded by.
 #' @param special_countries character: A string with 3-letter country codes.
 #'
+#' @return list
 #' @export
 db_create_coverage_table <- function(ref_year_table,
                                      pop_table,
+                                     cl_table,
+                                     incgrp_table,
                                      ref_years,
                                      digits = 2,
                                      special_countries =
-                                       c("ARG", "CHN", "IDN", "IND")) {
+                                       c("ARG", "CHN", "IDN", "IND", "SUR")) {
```

Comment:
2 thougts on this:
1) It may be worth finding a more explicit name. These countries could be special in many ways. Something like `countries_to_include` maybe?
2) We should start removing hard coded values from our codebase, and move it to a dedicated file in the pipeline.

Reply:
I agree. These are basically the countries where U/R surveys are used for the line-up and coverage should thus be based on U/R population numbers. Maybe something like `urban_rural_countries` , `non_national_pop_countries` ? 

I also agree with 2) but we don't really have a good system in place for that. I suggest keeping such improvements separate from the current PR.

Reply:
Renamed to `urban_rural_countries`. Let me know what you think